Jenkinsfile_parameterized

```diff
@@ -180,6 +180,10 @@ def initialiseLogstashConfFiles(KeyVault keyVault) {
         sh "cp packer_images/dead_letter_indexing_pipeline.conf.in packer_images/dead_letter_indexing_pipeline.conf"
         sh "sed -i 's|\"\${ES_DATA_NODES_URL}\"|${es_url}|' packer_images/dead_letter_indexing_pipeline.conf"
         sh "cat packer_images/dead_letter_indexing_pipeline.conf"
+
+        sh "cp packer_images/sweeper_dead_letter_indexing_pipeline.conf.in packer_images/sweeper_dead_letter_indexing_pipeline.conf"
+        sh "sed -i 's|\"\${ES_DATA_NODES_URL}\"|${es_url}|' packer_images/sweeper_dead_letter_indexing_pipeline.conf"
+        sh "cat packer_images/sweeper_dead_letter_indexing_pipeline.conf"
```

Comment:
rather than duplicating this, we could introduce a variable which resolves to one of the 2 files and then use that var. i.e.:
```dead_letter_filename = ${productName==sweeper?  sweeper_dead_letter_indexing_pipeline.conf.in : dead_letter_indexing_pipeline.conf.in}sh "sed -i 
      s|\"\${ES_DATA_NODES_URL}\"|${es_url}|' packer_images/${dead_letter_filename}"
      sh "cat packer_images/${dead_letter_filename}"
```
(pseudocode not verified)